packages/applications/legacy/src/modules/project/eventHandlers/handleProjectRawDataImported.ts

```diff
@@ -46,4 +50,48 @@ export const handleProjectRawDataImported =
     if (res.isErr()) {
       console.error('handleProjectRawDataImported error', res.error);
     }
+
+    if (classe === 'Classé' && featureFlags.SHOW_GARANTIES_FINANCIERES) {
+      const typeGarantiesFinancières =
+        data.garantiesFinancièresType &&
+        convertGarantiesFinancièresType(data.garantiesFinancièresType);
+
+      if (typeGarantiesFinancières) {
+        const identifiantProjetValue = IdentifiantProjet.convertirEnValueType(
+          `${appelOffreId}#${periodeId}#${familleId}#${numeroCRE}`,
+        ).formatter();
+
+        try {
+          await mediator.send<GarantiesFinancières.ImporterTypeGarantiesFinancièresUseCase>({
+            type: 'Lauréat.GarantiesFinancières.UseCase.ImporterTypeGarantiesFinancières',
+            data: {
+              identifiantProjetValue,
+              importéLeValue: new Date(event.occurredAt).toISOString(),
+              typeValue: typeGarantiesFinancières,
+              ...(data.garantiesFinancièresDateEchéance &&
+                GarantiesFinancières.TypeGarantiesFinancières.convertirEnValueType(
+                  typeGarantiesFinancières,
+                ).estAvecDateÉchéance() && {
+                  dateÉchéanceValue: new Date(data.garantiesFinancièresDateEchéance).toISOString(),
+                }),
+            },
+          });
+        } catch (error) {
+          logger.error(
+            `handleProjectRawDataImported : enregistrer le type de garantie financière (projet ${identifiantProjetValue}) : ${error.message}`,
+          );
+        }
+      }
+    }
   };
+
+const convertGarantiesFinancièresType = (typeImporté: string) => {
```

Comment:
J'aurais plutôt nommé la fonction avec le terme "convertir" pour garder une cohérence avec le reste 

```suggestion
const convertirGarantiesFinancièresType = (typeImporté: string) => {
```